core/src/main/java/hudson/model/UpdateSite.java

```diff
@@ -160,14 +160,27 @@ public long getDataTimestamp() {
         return dataTimestamp;
     }
 
+    /**
+     * Update the data file from the given URL if the file
+     * does not exist, or is otherwise due for update.
+     * Accepted formats are JSONP or HTML with {@code postMessage}, not raw JSON.
+     * @return null if no updates are necessary, or the future result
+     * @since TODO
+     */
+    public @CheckForNull Future<FormValidation> updateDirectly() {
+        return updateDirectly(DownloadService.signatureCheck);
+    }
+
     /**
      * Update the data file from the given URL if the file
      * does not exist, or is otherwise due for update.
      * Accepted formats are JSONP or HTML with {@code postMessage}, not raw JSON.
      * @param signatureCheck whether to enforce the signature (may be off only for testing!)
      * @return null if no updates are necessary, or the future result
      * @since 1.502
+     * @deprecated use {@linkplain #updateDirectly()}
      */
+    @Deprecated
```

Comment:
that would break source compatibility, happy to do that, but that could break some (at least internal) workflows, and goes against the Jenkins API deprecation policy.  Lets at least deprecate it for at least one LTS version first (could we then start a sweep of all deprecated API and look at restrict them?)

Reply:
Fine with me. We break compatibility too often in recent releases